Dockerfile

```diff
@@ -32,7 +32,9 @@ USER root
 RUN mkdir -p /.config/rhoas && chmod 777 -R /.config/rhoas && echo "{}" > /.config/rhoas/config.json && chmod 777 /.config/rhoas/config.json
 RUN mkdir /.kube && chmod 777 /.kube
 USER rhoas
+WORKDIR /home/rhoas
 
+COPY scripts /home/rhoas/scripts
```

Comment:
I've added the scripts already, though not to this location: https://github.com/redhat-developer/app-services-tools/blob/main/Dockerfile#L36